inference-engine/ie_bridges/c/samples/object_detection_sample_ssd/main.c

```diff
@@ -69,11 +76,15 @@ int ParseAndCheckCommandLine(int argc, char *argv[]) {
     if (help)
         return -1;
     if (input_model == NULL) {
-        printf("Model is required but not set. Please set -m option. \n");
+        printf("Model is required but not set. Please set -m option.\n");
         return -1;
     }
     if (img_msg == NULL) {
-        printf("Input is required but not set.Please set - i option.\n");
+        printf("Input is required but not set.Please set -i option.\n");
+        return -1;
+    }
+    if (custom_plugin_cfg_msg == NULL && device_name != "CPU") {
```

Comment:
Looks like the sample cannot be used for GPU without custom kernels anymore

Reply:
reverted due to misunderstanding